libs/sdk-ui-filters/src/AttributeFilter/AttributeDropdown/AttributeDropdownList.tsx

```diff
@@ -55,6 +58,8 @@ export const AttributeDropdownList: React.FC<IAttributeDropdownListProps> = ({
         return isNonEmptyListItem(i) ? (isSelectionByUri ? i.uri : i.title) : "empty";
     }, []);
 
+    const itemHeight = useMemo(() => (isMobile ? MOBILE_LIST_ITEM_HEIGHT : ITEM_HEIGHT), [isMobile]);
```

Comment:
Nitpick: there is no need to memoize this, the result is always a number (= value type) and the computation is "cheaper" than comparing the useMemo dependencies.